ros_cross_compile/ros_cross_compile.py

```diff
@@ -189,10 +198,17 @@ def cross_compile_pipeline(
         skip_rosdep_keys,
         custom_rosdep_script,
         custom_data_dir,
-        custom_setup_script)
+        custom_setup_script,
+        args.runtime_tag)
+
+    skip = set(args.stages_skip)
+
+    # Only package the runtime image if the user has specified a tag for it
+    if not args.runtime_tag:
```

Comment:
Since `arg.runtime_tag` has a default value, will this ever be true?

Reply:
Oops, good catch, it was supposed to deafult to None